packages/data-stores/src/site/controls.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Internal dependencies
+ */
+import { wpcomRequest, WpcomClientCredentials } from '../utils';
+import { ActionType, CreateSiteAction } from './types';
+
+export default function createControls( clientCreds: WpcomClientCredentials ) {
+	return {
+		[ ActionType.CREATE_SITE ]: async ( action: CreateSiteAction ) => {
+			const { authToken, ...providedParams } = action.params;
+			const defaultParams = {
```

Comment:
I know this is WIP, but in my local attempt I unashamedly plagiarized your `User` control method style:
			
```

		    // Creating a site requires authentication. Let's make sure the proxy iframe
            // has set the required cookie.
            reloadWpcomProxy();

           const defaultParams = {
				client_id: clientCreds.client_id,
				client_secret: clientCreds.client_secret,
                // will find an available `*.wordpress.com` url based on the `blog_name`
				find_available_url: true,
			};
			
			const mergedParams = {
			    ...defaultParams,
                ...action.params,
                // Set to false because site validation should be a separate action
                validate: false,
                // Private site is default setting
                public: -1,
            };
			
			const newSite = await wpcomRequest( {
				path: '/sites/new',
				apiVersion: '1.1',
				method: 'post',
				body: mergedParams,
			} );
```

And this is what we'd pass it from Gutenboarding:

```
wp.data.dispatch( 'automattic/sites' ).createSite( {
                blog_name: 'batman is my neighbour',
                blog_title: 'superman does not exist',
                locale: 'en',
                options: {
                    theme: "pub/maywood",
                    use_theme_annotation: true, // used by FSE to headstart template-based sites
                    site_segment: 1, // we only have annotations for `1` when use_theme_annotation is `true` 
                    site_vertical: "p31v28",
                    site_vertical_name: "Cameras & Photography",
                    site_information: { title: "hello" },
                    site_creation_flow: "gutenboarding",
                },
            }  );
```

Reply:
> 'batman is my neighbour'

I'd probably read that blog 😆

Reply:
🦇👨